src/app/admin/components/reports/reports.directive.js

```diff
@@ -405,21 +392,15 @@
                     activity.certificationEdition = data[i].newData.certificationEdition.name;
                     activity.certificationDate = data[i].newData.certificationDate;
                     activity.friendlyCertificationDate = new Date(activity.certificationDate).toISOString().substring(0, 10);
-                    questionable = data[i].activityDate > data[i].newData.certificationDate + (vm.questionableRange * 24 * 60 * 60 * 1000);
                     activity.details = [];
                     var statusChange = nestedCompare(data[i].originalData, data[i].newData, 'certificationStatus', 'name', 'Certification Status');
                     if (statusChange) {
                         var statusActivity = angular.copy(activity);
                         statusActivity.details = statusChange;
                         output.status.push(statusActivity);
-
-                        // count status change as questionable
-                        activity.details.push('<span class="bg-danger"><strong>' + statusChange + '</strong></span>');
-                        activity.questionable = true;
                     }
                     if (data[i].newData.certificationEdition.name === '2011') {
                         activity.action = '<span class="bg-danger">' + activity.action + '</span>';
```

Comment:
Giving things the class `bg-danger` is how things are depicted as being questionable. This line indicates an activity is questionable on the screen, though we're not parsing that any more

Reply:
So that questionable flag on the activity object should still be in there?

Reply:
No, we're getting rid of the questionable flag, but I'm looking at line 421, where the action is highlighted using the class "bg-danger"